WORKSPACE

```diff
@@ -92,28 +92,28 @@ http_archive(
 # Disk images
 http_file(
     name = "alpine_image",
-    sha256 = "a90150589e493d5b7e87297056b6e124d8af1b91fa2eb92bab61a839839e287b",
+    sha256 = "f87a0fd3ab0e65d2a84acd5dad5f8b6afce51cb465f65dd6f8a3810a3723b6e4",
     urls = [
-        "https://dl-cdn.alpinelinux.org/alpine/v3.16/releases/x86_64/alpine-virt-3.16.3-x86_64.iso",
-        "https://storage.googleapis.com/builddeps/a90150589e493d5b7e87297056b6e124d8af1b91fa2eb92bab61a839839e287b",
```

Comment:
I am pretty sure this is not the alpine that ends up being used in tests. check out https://github.com/kubevirt/kubevirt/commit/bc73991cd9b79132df909557f39e5f94cd1926c8

Reply:
See commit ddd0c3e85dd5415f8a21d04e97e324dc3ae5d7be.

Reply:
oh I didn't realize we have two variations

Reply:
according to @andreabolognani though, 3.19 (alpine-test-tooling) was just fine virtio-mem-wise?

Reply:
as far as I know it was only enabled in the lts flavor https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/61693.

I'd just use the latest alpine, which I tested manually.

Reply:
If you scroll down a little in that merge request, it made it there too

Reply:
ok, but I don't think this should block this PR.

Reply:
Up to you